src/Resources/views/Block/block_search_result.html.twig

```diff
@@ -12,7 +12,7 @@ file that was distributed with this source code.
 {% extends sonata_block.templates.block_base %}
 
 {% block block %}
-    {% set show_empty_boxes = sonata_admin.adminPool.container.getParameter('sonata.admin.configuration.global_search.empty_boxes') %}
+    {% set show_empty_boxes = sonata_admin.adminPool.getOption('sonata.admin.configuration.global_search.empty_boxes') %}
```

Comment:
This should be fixed in `3.x`, [`Pool::getContainer` was deprecated](https://github.com/sonata-project/SonataAdminBundle/pull/6475), I didn't check templates thinking that there shouldn't be calls to the container.

Given this is a block, the solution could be to inject to that block the parameter and pass it to the twig template because I don't think `sonata.admin.configuration.global_search.empty_boxes` is available through `Pool::getOption` and shouldn't be.

Reply:
You are right, I hadn't tested whether the parameter is getting through or not and it's not, it's failing silently.

Reply:
Can one of you open the PR on 3.x to fix the deprecation ?

Reply:
I made https://github.com/sonata-project/SonataAdminBundle/pull/6593